crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs

```diff
@@ -143,11 +143,12 @@ impl Context {
                 let result_ids = dfg.instruction_results(instruction_id);
                 if Self::value_is_array_address(result_ids[0], dfg) {
                     self.track_array_address(result_ids[0], binary, dfg);
-                    return;
+                    (Vec::new(), Vec::new())
+                } else {
+                    let result_acir_var = self.convert_ssa_binary(binary, dfg);
+                    self.ssa_value_to_acir_var.insert(result_ids[0], result_acir_var);
```

Comment:
Is this extra line needed?

Reply:
Shouldn't it do that when we return the vectors?

Reply:
Good spot. Removed

Reply:
Oh, wait, bad fix

Reply:
Reverted, and deleted the correct line this time